tests/utils/test_cache_utils.py

```diff
@@ -713,3 +716,210 @@ def test_hybrid_cache_exportability(self):
             dynamic_shapes=dynamic_shapes,
             strict=False,
         )
+
+
+class SyntheticCacheTest(unittest.TestCase):
+    """
+    Synthetic tests for StaticCache, SlidingWindowCache, and HybridCache.
+    Uses window_size=4, max_cache_len=4 for all tests.
```

Comment:
We might change these details in the future, so I'd recommend a more future-proof docstring (e.g. "Tests using dummy data inputs to test cache behaviour")